Add info box component

stephan281094 opened the pull request:
Before:
<img width="1183" alt="Screen Shot 2021-08-16 at 15 22 49" src="https://user-images.githubusercontent.com/4006802/129570650-dd367d1a-163a-4550-9aed-1ce85d50b4c9.png">

After:
<img width="897" alt="Screen Shot 2021-08-16 at 15 22 33" src="https://user-images.githubusercontent.com/4006802/129570696-a9b07ecb-7140-423e-87ce-fb8e5ce14353.png">

This pull request changes the following:
- Add info box component.
- Update "suggest an idea" section of the projects page to use this component.
- Replace the service component on the services page with this component.

Having some difficulties running existing tests. Jest doesn't seem to load the `.env` file in order to initialize the Contentful API. I managed to fix this locally by installing dotenv and updating the `yarn test` script, but it still resulted in a snapshot mismatch. Likely because the snapshot is outdated.

Also noticed a few issues:
- We're using camelCase, kebab-case AND PascalCase for file names. Can we please use something consistently?
- Same applies to asset file names. Would be nice to have these be more consistent.
- Interfaces and types are both used for defining prop types. Again, inconsistent.
- The components folder is now separated into decoration and layout. Why not put components directly inside the components folder? These subfolders don't add any value in my opinion.

JoaquinTrinanes commented:
> Also noticed a few issues:
> 
>     * We're using camelCase, kebab-case AND PascalCase for file names. Can we please use something consistently?
All files inside the `pages` folder are kebab-case because Next.js creates the routes based on their name. All other ones should be camelCase, some may have slipped, I'll rename them.

>     * Same applies to asset file names. Would be nice to have these be more consistent.
We should, in this case we're simply copy-pasting the files as-is. We should try to come up with a naming convention before there are too many assets.
>     * Interfaces and types are both used for defining prop types. Again, inconsistent.
The only `type` data declaration I can find is in the `hello.ts` file, which came with the project barebones structure. I'll delete it. Other than that we shouldn't use type unless aliasing something. I'll add an ESLint rule for that.
>     * The components folder is now separated into decoration and layout. Why not put components directly inside the components folder? These subfolders don't add any value in my opinion.
When I created those the idea was that one would hold a page layout (for example, all pages have the navbar and footer, the about page has separated sections...) while the decoration folder would hold generic components, like buttons and the pixel sprites. I don't know if it makes any sense, and if it's the use they have right now.

stephan281094 commented:
> All files inside the `pages` folder are kebab-case because Next.js creates the routes based on their name. All other ones should be camelCase, some may have slipped, I'll rename them.

There were only a few files, I updated them to now use camel case.

> The only `type` data declaration I can find is in the `hello.ts` file, which came with the project barebones structure. I'll delete it. Other than that we shouldn't use type unless aliasing something. I'll add an ESLint rule for that.

I guess I was just unlucky when briefly checking other files and seeing the inconsistency. I replaced the only `type` I found with an `interface`.

> When I created those the idea was that one would hold a page layout (for example, all pages have the navbar and footer, the about page has separated sections...) while the decoration folder would hold generic components, like buttons and the pixel sprites. I don't know if it makes any sense, and if it's the use they have right now.

That makes sense. Do you want me to move the info box component into another folder?
